added queue tweaks after ribgen, changed frame task id to add _f###

mnemyx opened the pull request.

jerrytessendorf commented:
Please don't add the f at the end.  It is unnecessary.

-jerry

> On Jul 16, 2015, at 8:46 AM, Gina <notifications@github.com> wrote:
> 
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/Clemson-DPA/dpa-pipe/pull/41
> 
> Commit Summary
> 
> added queue tweaks after ribgen, changed frame task id to add _f###
> File Changes
> 
> M dpa/ui/dk/maya.py (12)
> Patch Links:
> 
> https://github.com/Clemson-DPA/dpa-pipe/pull/41.patch
> https://github.com/Clemson-DPA/dpa-pipe/pull/41.diff
> —
> Reply to this email directly or view it on GitHub.
>

mnemyx commented:
Removed

On Thu, Jul 16, 2015 at 8:48 AM, jerrytessendorf <notifications@github.com>
wrote:

> Please don't add the f at the end. It is unnecessary.
>
> -jerry
>
> > On Jul 16, 2015, at 8:46 AM, Gina <notifications@github.com> wrote:
> >
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/Clemson-DPA/dpa-pipe/pull/41
> >
> > Commit Summary
> >
> > added queue tweaks after ribgen, changed frame task id to add _f###
> > File Changes
> >
> > M dpa/ui/dk/maya.py (12)
> > Patch Links:
> >
> > https://github.com/Clemson-DPA/dpa-pipe/pull/41.patch
> > https://github.com/Clemson-DPA/dpa-pipe/pull/41.diff
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Clemson-DPA/dpa-pipe/pull/41#issuecomment-121947331>.
>
